db-checkbook/build_scripts/02_build.py

```diff
@@ -1,5 +1,8 @@
 import pandas as pd
 
+def run_categorization(df):
```

Comment:
best practice might be to open a separate PR for changes made to the build script and keep the scope of this PR as small as possible (limiting to dataloading)